Core/BossPrototype.lua

```diff
@@ -1464,6 +1464,8 @@ end
 -- @param[opt] icon the bar icon (spell id or texture name)
 function boss:Bar(key, length, text, icon)
 	local textType = type(text)
+	local bars = core:GetPlugin("Bars")
+	if bars and not bars.db.profile["cast"] and text == L.cast then return end
```

Comment:
Yeah totally my fault for not checking, I forgot `text` here is after formatting.

How would you handle different locales for L.cast? Would a per-ability per-encounter tickbox option be better than a generic catch-all in boss:Bar?